Samples/Widgets/MultipleWidgets/cpp-win32-packaged/SampleWidgetProviderApp/WidgetProvider.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "WidgetProvider.h"
+#include "WidgetImpl.h"
+
+typedef std::function<std::shared_ptr<WidgetImpl>(winrt::WidgetContext, winrt::hstring)> CreateWidgetImplFn;
```

Comment:
use using instead of typedef to ease reading

Reply:
chaged to typedef